src/ar_http_iface_cowboy_handler.erl

```diff
@@ -185,12 +185,13 @@ do_handle(<<"POST">>, [<<"arql">>], Req) ->
 
 %% @doc Return the data field of the transaction specified via the transaction ID (hash) served as HTML.
 %% GET request to endpoint /tx/{hash}/data.html
-do_handle(<<"GET">>, [<<"tx">>, Hash, << "data.", _/binary >>], Req) ->
+do_handle(<<"GET">>, Addr = [<<"tx">>, Hash, << "data.", _/binary >>], Req) ->
```

Comment:
Starting from #144 I think it should be only the functionality of the gateway server so it probably makes sense to wait until it's merged and move it there.